### DISCUSSION TITLE
Image-based Lighting

### DISCUSSION THREAD

**Discussion by WdsPjm** [vsg-dev/VulkanSceneGraph#1079] [General]
Is the IBL functionality already implemented in VSG?

**Comment by robertosfield** [comment]
Not yet.

Is there a specific approach you had in mind?

**Comment by WdsPjm** [comment]
I need to use this in my project. I'll give it a try and see if I can implement it myself.

**Comment by robertosfield** [comment]
On my TODO list for this year I have Image Based Lighting on it, but won't tackle it near term.  Happy for others to jump in and come up with a solution.  I haven't looked at the topic in depth enough to know which approach would be best for the VSG. 

Happy for others to dive in an come up solutions.  If open sourced then perhaps a example that implements it then we can look at that as source of inspiration for integrating it into the core VSG/ShaderSets.
